charter/index.md

```diff
@@ -0,0 +1,215 @@
+---
+title: Font and Text Community Group Charter
+---
+This charter:  {{ page.url | remove: "index.html" | prepend: site.url }}
+
+Feedback about this document:
+    https://github.com/font-text/font-text.github.io/issues
+
+## Scope
+
+The Font and Text community group gathers individuals and organizations
+interested in discussing and developing specifications and implementations
+for technologies such as shaping and layout
+which operate on and at the interface between
+text encoding and font formats.
+
+Under the present charter, the Community Group will
+determine a structure and process
+under which such specifications may be created
+by which the greatest number of interested parties may be able to participate.
+Until such a structure and process is determined,
+no technical development of specifications or implementations
+will be undertaken by this Community Group
+and no technical contributions will be accepted.
+
+## Deliverables
+
+* This Community Group aims to produce a proposed Charter under which
+this community should conduct its technical work,
+either to be used as an updated Charter for this CG,
+or to set up a different group,
+possibly a W3C Working Group,
+to conduct that work.
+
+* If the Community Groups determines that it would be appropriate
+to divide the technical work into several venues,
+it may produce several such proposed charters
+rather than a single one.
```

Comment:
Thinking "aloud," the idea of rewriting the charter (potentially repeatedly) to enable technical work seems a bit awkward. I also wonder if charter updates might require some of us to refresh corporate approvals.

I wonder if it might make sense for the CG to have a stable charter around being a discussion forum that aims to spawn new groups (typically new WG or CG with the interested subset of the parent CG membership) to perform specific technical work.

To give a concrete example, I am eager to explore creation of a WG chartered to specify 32 bit gid & offset support with a clear path to the changes landing in OpenType, OFF, or both. 

I'm curious what others think about this?

Reply:
Isn’t this covered by the part you quoted? The proposed draft gives us the flexibility to do this. Any kind of technical contribution will need reapprovals; that’s just unavoidable if we want to make progress.

Reply:
IIUC we avoid reapproval if we plan to spawn new, separate, groups - my guess is usually a subset of the CG - to do technical work instead of rechartering this CG.

Reply:
I lean in favor of spawning new groups, and generally I would prefer WGs instead of CGs, but I don't think we really need to commit to any particular approach at this point. The phrasing of this bullet and the one before, to my reading, allows either, and I think that's fine.

Reply:
I agree, but anything we can do to ease the path for people having to satisfy lawyers is worth doing. Let's turn things around and *prioritize* spawning technical groups, so that people don't have to get re-approval to stay in the current CG.

Reply:
@rsheeter Is this concern revolved your you in the current text of the draft? The goals of this group (particularly *not* having technical specifications as part of our deliverables) vs. and the intention to spawn technical group(s) or working group(s) seems to be spelled out quite a bit more clearly than at the time you commented on this.

I'm going to mark this thread as resolved, but if my impression is wrong and you have an ongoing concern about the current text please do open another comment. We're in the final 10 days before this charter is ratified unless there are new objections raised.